script.js

```diff
@@ -1 +1,89 @@
+import { renderOnCreate, renderAllData, renderFilteredData } from "./render.js";
 
+document.addEventListener('DOMContentLoaded', () => renderAllData(""));
```

Comment:
Since we can use default argument to be "" in case no argument is provided. We can write like this now

```suggestion
document.addEventListener('DOMContentLoaded', () => renderAllData());
```